CONTRIBUTING.md

```diff
@@ -34,7 +34,8 @@ Or run tests in watch mode:
 npm test --watch
 ```
 
-By default the tests are executed in your local Devnet. If you want to use a specific rpc node, you have to set some global variables before executing the tests :
+By default the tests are executed in your local Devnet. If you want to use a specific
```

Comment:
@tabaktoni @ivpavici We can also remove this completely and just rely on the docker approach. By doing that, we do not confuse the contributor with other options that can easily break